### PR TITLE
Use shader_cache 1.1.0 in artefacts

### PR DESCRIPTION
**Pull request by dcvz**

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - dev
-  pull_request_target:
+  pull_request:
```
Comment (dcvz) [inline .github/workflows/validate.yml]: Used to test this pipeline, to be reverted before merge.

File: .github/workflows/validate.yml [hunk 2]
```diff
@@ -90,7 +91,7 @@ jobs:
         run: |
           mv cmake-build/Zelda64Recompiled Zelda64Recompiled
           rm -rf assets/scss
-          tar -czf Zelda64Recompiled-${{ runner.os }}-${{ matrix.type }}.tar.gz Zelda64Recompiled assets/
+          tar -czf Zelda64Recompiled-${{ runner.os }}-${{ matrix.type }}.tar.gz Zelda64Recompiled assets/ shadercache/
```
Comment (Mr-Wiseguy) [inline .github/workflows/validate.yml]: The shader cache is embedded into the binary, so there's no need to copy it into the final archive

File: .github/workflows/validate.yml [hunk 3]
```diff
@@ -173,3 +175,4 @@ jobs:
             dxil.dll
             SDL2.dll
             assets/
+            shadercache/
```
Comment (Mr-Wiseguy) [inline .github/workflows/validate.yml]: Ditto from the other comment.

File: .github/linux/appimage.sh [hunk 4]
```diff
@@ -5,6 +5,7 @@ chmod a+x linuxdeploy*
 mkdir -p AppDir/usr/bin
 cp Zelda64Recompiled AppDir/usr/bin/
 cp -r assets/ AppDir/usr/bin/
+cp -r shadercache/ AppDir/usr/bin/
```
Comment (Mr-Wiseguy) [inline .github/linux/appimage.sh]: Same deal as the other comments

**Review by Mr-Wiseguy — CHANGES_REQUESTED** [review]

**Review by Mr-Wiseguy — APPROVED** [review]

**Pull request merged**